documentation/book/con-resource-kafka-connect.adoc

```diff
@@ -0,0 +1,93 @@
+[id='kafka-connect-resource-{context}']
+= Kafka Connect
+
+The operator watches for `KafkaConnect` resource in order to find and get configuration for a Kafka Connect cluster to deploy.
+
+The `KafkaConnectS2I` resource provides configuration for a Kafka Connect cluster deployment using Build and Source2Image features and works only with {OpenShiftName}.
+
+Labels that are applied to the `KafkaConnect` or `KafkaConnectS2I` resources will also be applied to the {ProductPlatformName} resources making up the Kafka Connect cluster. This provides a convenient mechanism for those resource to be labelled in the users preferred way.
+
+The `KafkaConnect` resource supports the following within its `spec`:
+
+.KafkaConnect Parameters
+[cols="30%,50%,20%",options="header"]
+|===
+|Parameter
+|Description
+|Default Value
+
+|`nodes`
+|Number of Kafka Connect worker nodes.
+|1
+
+|`image`
+| The Docker image to be used by the Kafka Connect workers. The default value is determined by the value specified in the  xref:STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE[`STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE`] environment variable of the Cluster Operator. If S2I is used on {OpenShiftName}, then it should be the related S2I image.
+|
+
+|`healthcheck-delay`
```

Comment:
The `healthcheck-delay` and `healthcheck-timeout` should be replaced by something like this:

livenessProbe.initialDelaySeconds
The initial delay for the liveness probe for each Kafka broker node. Default is 15.

livenessProbe.timeoutSeconds
The timeout on the liveness probe for each Kafka broker node. Default is 5.

readinessProbe.initialDelaySeconds
The initial delay for the readiness probe for each Kafka broker node. Default is 15.

readinessProbe.timeoutSeconds
The timeout on the readiness probe for each Kafka broker node. Default is 5.